net/socks5/socks5_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package socks5 is a SOCKS5 server implementation
```

Comment:
no package docs in tests

you could add a file comment if you wanted. (with a blank line above the package line)

Reply:
Removed package doc.

I think a file comment would just say that it is tests for SOCKS5, which isn't useful.